Autopilot/PathManager/Inc/landingTakeoffManager.hpp

```diff
@@ -0,0 +1,170 @@
+        */
+        static double approachSpeed(bool ifPackage);
+
+        /* 
+            This function takes the stall speed and multiplies it by a constant to return a slow flight speed
+        
+            @param bool * ifPackage - this boolean holds if there is a package on board the aircraft
+
+            @return double - this function will return the slow flight speed for the aircraft
+        */
+        static double slowFlightSpeed(bool ifPackage);
+
+        /*
+            This function takes the coordinates of the landing spots and direction of landing
+            Creates stopping points, aiming point, and intersection point
+
+            @param Telemetry_PIGO_t * input - this variable will hold the stopping point information, which is passed in through telemetry
+            @param double * currentAltitude - this variable holds the current altitude of the aircraft, which is used for the altitude of the intersection point
+
+            @return _LandingPath - this function will return a structure that contains the GPS locations of the stopping, aiming, and intersection points
+        */
+        static _LandingPath createSlopeWaypoints(const Telemetry_PIGO_t & input, float currentAltitude);
+
+        /*
+            This function translates waypoint output data into landing takeoff output data
+            Takes in a waypoint data out structure, reformats it, and outputs a landingTakeoffOutput instance
+
+            @param _WaypointManager_Data_Out * outputData - this variable will hold the output data that is outputted by waypointManager functions
+
+            @return _LandingTakeoffOutput - this function will output a landing and takeoff output struct
+        */
+        static _LandingTakeoffOutput translateWaypointCommands(const _WaypointManager_Data_Out & outputData);
+
+        /*
+            This function translates landing takeoff data to output data
+            Takes in a landingTakeoffOutput instance, translates its information into the 2 types of coordinated turn inputs
+
+            @param _LandingTakeoffOutput* outputData - this pointer points to the output of the landing and takeoff states
+            @param SFOutput_t* sensorOutput - this pointer points to the sensor output of sensorFusion
+            @param IMUOutput_t* imuOutput - this pointer points to the imu output of sensorFusion
+            @param CoordinatedTurnInput_t * turnInput - this pointer points to the coorinated turn input structure that needs to be changed
+            @param AltitudeAirspeedInput_t * altitudeAirspeedInput - this pointer points to the altitude and airspeed input structure that needs to be changed
+        */
+        static void translateLTSFCommandsToCoordTurns(const _LandingTakeoffOutput & outputData, const SFOutput_t & sensorOutput, const IMU_Data_t & imuOutput, CoordinatedTurnInput_t & turnInput, AltitudeAirspeedInput_t & altitudeAirspeedInput);
```

Comment:
This function and the one above are common to both takeoff and landing, right ? Would it be wise to add another comment seperation section to indicate that it's common ?